packages/block-library/src/buttons/edit.native.js

```diff
@@ -31,8 +34,9 @@ function ButtonsEdit( {
 	onAddNextButton,
 	shouldDelete,
 	isInnerButtonSelected,
+	setAttributes,
 } ) {
-	const { align } = attributes;
+	const { contentJustification } = attributes;
```

Comment:
You can move this destructuring to the existing destructure for the entire function, e.g.:

```js
function ButtonsEdit( {
	isSelected,
	attributes: { contentJustification },
	onDelete,
	onAddNextButton,
	shouldDelete,
	isInnerButtonSelected,
	setAttributes,
} ) {
```

Reply:
Thanks, updated! somehow I had an older version